components/worker/.env.example

```
@@ -97,4 +97,20 @@ CRYPTO_ENCRYPT_SECRET_KEY_PLUGIN_CRM=CHANGE_ME
+# Tenant Manager service URL (required when MULTI_TENANT_ENABLED=true)
+#MULTI_TENANT_URL=http://tenant-manager:8080
+# Environment for cache key segmentation
+#MULTI_TENANT_ENVIRONMENT=staging
+# Soft limit for tenant connection pools (LRU eviction)
+#MULTI_TENANT_MAX_TENANT_POOLS=100
+# Seconds before idle tenant connection is eviction-eligible
+#MULTI_TENANT_IDLE_TIMEOUT_SEC=300
+# Consecutive failures before circuit breaker opens
+#MULTI_TENANT_CIRCUIT_BREAKER_THRESHOLD=5
+# Seconds before circuit breaker resets (half-open)
+#MULTI_TENANT_CIRCUIT_BREAKER_TIMEOUT_SEC=30
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)